### PR TITLE
Remove sendopcache. Switch to php 5.6.

### PR DESCRIPTION
**Pull request by denisshevchenko**

### CODE REVIEW

File: sdk.nix [hunk 1]
```diff
@@ -170,11 +170,7 @@ rec {
   };
 
   phpPackages = {
-    inherit (pkgs.phpPackages) memcache memcached redis zmq zendopcache xdebug;
+    inherit (pkgs.phpPackages) memcache memcached redis zmq xdebug;
     inherit (ugpkgs) newrelic-php imagick;
   };
-
-  perlPackages = {
```
Comment (proger) [inline sdk.nix:177]: stray chunk?

Comment (phunehehe) [inline sdk.nix:177]: @denisshevchenko I don't think you intended to remove this?

Comment (denisshevchenko) [inline sdk.nix:177]: Nope... I just copied `sdk.nix` from `eris/deps/microgram` and change `php56`. Should I back that `perlPackages`?

Comment (phunehehe) [inline sdk.nix:177]: well... you shouldn't make any change you don't intend to make :)

**Pull request closed without merging**